packages/cml/src/replaceCmlPlaceholders/replaceCmlPlaceholders.ts

```diff
@@ -78,6 +78,16 @@ const replacePropValues = (mdx: string, values: Record<string, string> = {}): st
       case 'number':
         searchValue = match;
         newValue = `{${propValue}}`;
+        break;
+      case 'json':
+        searchValue = match;
+
+        if (propValue && typeof propValue === 'object') {
+          newValue = `{${JSON.stringify(propValue)}}`;
```

Comment:
I would like to set it as an object instead because I suppose that setting it as a string, the consumer component will have to do a `JSON.parse` in order to access the json data.

But I haven't found a way to pass it as an object.

WDYT? Any suggestions?

Reply:
can we use the `flatObject` as in the line 70?

Reply:
This is what I've got with `flatObject`. 

`<Text prop=json=\"value\"/>`
That syntax is weird

Is that expected?

<img width="736" alt="Screen Shot 2023-07-06 at 16 49 10" src="https://github.com/cmpsr/composer/assets/1816129/bbb50383-1cc8-41bc-83c8-129c4cfda806">

Reply:
tested and it works as it is